wgsl/index.bs

```diff
@@ -4026,7 +4066,7 @@ The last column in the table below uses the format-specific
   <tr><td>rgba32uint<td>32uint<td>r, g, b, a<td>vec4&lt;u32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
   <tr><td>rgba32sint<td>32sint<td>r, g, b, a<td>vec4&lt;i32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
   <tr><td>rgba32float<td>32float<td>r, g, b, a<td>vec4&lt;f32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
-  <tr><td>bgra8unorm<td>8unorm<td>b, g, r, a<td>vec4&lt;f32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>bgra8unorm<td>8unorm<td>b, g, r, a<td>vec4&lt;f32&gt;(CTF(b), CTF(g), CTF(r), CTF(a))
```

Comment:
Isn't this a double-swap?    The 3rd column shows the channel order already.

Reply:
I can revert this, but I thought `.r` would be the `b` channel of bgra8unorm.